dmscripts/export_framework_results_reasons.py

```diff
@@ -194,6 +161,14 @@ def create_row(self, record):
             contact_details(record)
 
 
+def get_questions_numbers_from_framework(framework_slug, content_loader):
```

Comment:
Not running the tests - they're OK.  It's running the script itself for G9 that breaks.